### ISSUE TITLE
Add More Live Examples for Molecule Visualization

### ISSUE THREAD

**Issue by TharunKumarrA** [TharunKumarrA/Molecule-Visualiser-DSA#8]
## Description
Currently, the application supports a limited number of live examples: **benzene**, **water**, **methane**, and **ethane**. To enhance the usability and testing capabilities, more molecular examples need to be added. These should include molecules with varying hybridizations and complexities.

## Acceptance Criteria
- Add new live examples for molecules of different types:
  - Molecules with double bonds (e.g., **Ethene (C₂H₄)**).
  - Molecules with triple bonds (e.g., **Acetylene (C₂H₂)**).
  - Molecules with **sp3d** and **sp3d2** hybridizations (e.g., **PCl₅**, **SF₆**).
- Ensure correct visualization of these examples in the 3D plot.
- Include a dropdown or selector in the UI to easily switch between examples. [Already present]

## Suggested Approach
1. **Frontend Updates:**
   - Add new options for the additional molecules in the dropdown menu for live examples.
   - Pass the selected molecule data to the molecule rendering logic.

2. **Backend Updates:**
   - Define the molecular structures for the new examples in the `EditMolecule.jsx` or the related logic.
   - Ensure correct bond angles and connectivity.

3. **Testing:**
   - Verify the 3D visualization for all examples, ensuring accuracy in geometry and bond representations.

---

### File: `/src/components/EditMolecule.jsx`
- **Add New Molecules:**
  - Extend the `sampleMolecules` array with additional molecules.
  - Define the molecular structures and ensure compatibility with existing logic.

---

### In `EditMolecule.jsx`
Add new molecules to the `sampleMolecules` array:
```javascript
const sampleMolecules = ["CH4", "C2H6", "C6H6", "H2O", "C2H4", "C2H2", "PCl5", "SF6"];
```
### Update the Molecule Construction Logic

```javascript
const handleShowMolecule = () => {
  switch (selectedMolecule) {
    case "C2H4": // Ethene
      // Define ethene structure with double bonds
      break;
    case "C2H2": // Acetylene
      // Define acetylene structure with triple bonds
      break;
    case "PCl5": // Phosphorus Pentachloride
      // Define PCl5 structure with sp3d hybridization
      break;
    case "SF6": // Sulfur Hexafluoride
      // Define SF6 structure with sp3d2 hybridization
      break;
    default:
      // Handle existing molecules
  }
};
```
### Suggested New Examples

- **Ethene (C₂H₄):** Molecule with double bonds.
- **Acetylene (C₂H₂):** Molecule with triple bonds.
- **Phosphorus Pentachloride (PCl₅):** Molecule with sp3d hybridization.
- **Sulfur Hexafluoride (SF₆):** Molecule with sp3d2 hybridization.
## Dependencies
- This issue depends on the completion of #2 and #3 before it can be worked on.

**Comment by adithya-menon-r** [comment]
Hey @TharunKumarrA, I would like to work on this issue when the dependent issues are solved.

**Comment by TharunKumarrA** [comment]
yes, @adithya-menon-r you can start working on this after the dependency issues are resolved, but as you can hold up only upto 2 issues at a time. I would suggest you to take up issues that you can work on now. Or if you are ok with it I can still assign you to this issue.

**Comment by adithya-menon-r** [comment]
Makes sense @TharunKumarrA. I'll work on other issues until the dependent ones are solved... Thanks!

**Comment by TharunKumarrA** [comment]
Yeah cool

**Comment by TharunKumarrA** [comment]
@adithya-menon-r can you take this issue now and provide some not too complex organic molecules (that have to some extent exact structures without any errors or overlaps) like with double bonds or with formal groups etc ...

**Comment by adithya-menon-r** [comment]
@TharunKumarrA I'll try to do it now... I was actually still working on issue #4... I don't think I added code to fix it yet in the PRs. Also, I pulled the merged changes and found a bug from the merge conflict resolution (a single missing closing bracket at line 371 in `Molecule.js`. Do I create an issue for it and create a PR?

Really sorry for all the confusing merge conflicts....

**Comment by TharunKumarrA** [comment]
No worries. The thing is the folder structure change made vercel deployment to fail. So I'm re deploying again. Not your fault tho. So I can't check now as I'm I'm travel. So if you can check locally pulling the main branch and see if there are any issues. If so just raise a pr with working screenshot l, maybe 1 is enough. I'll merge it @adithya-menon-r

**Comment by TharunKumarrA** [comment]
No need to create an issue just raise a pr
